src/arrays/ladderops.jl

```diff
@@ -37,7 +37,16 @@ function momentumop(n::Int)
     return scale(im/sqrt(2.), cop-cop')
 end
 
+squeeze_construct(a::AbstractQuMatrix,b::AbstractQuMatrix, z::Union(Complex64, Float64)) = 0.5*(z'*(a*b)-z*(a'*b'))
```

Comment:
I am not sure I understand this construct, but in QuDOS I have used the following
```
function squeezingop( nb::Int, z::Union(Complex128, Float64) )
	ad2 = 0.5*z*creationop( nb )*creationop( nb )
	sparse( expm( dense(ad2' - ad2) ) )
end
```
Do you think we will need the `squeezing` operation?